corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ontonotes/ConvertOntonotesToColumn.java

```diff
@@ -0,0 +1,160 @@
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Constituent;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.SpanLabelView;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.View;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.XmlTextAnnotation;
+import edu.illinois.cs.cogcomp.core.io.LineIO;
+import edu.illinois.cs.cogcomp.core.utilities.XmlDocumentProcessor;
+import edu.illinois.cs.cogcomp.core.utilities.XmlDocumentProcessor.SpanInfo;
+import edu.illinois.cs.cogcomp.nlp.corpusreaders.ereReader.CoNLL2002Writer;
+import edu.illinois.cs.cogcomp.nlp.corpusreaders.ontonotes.utils.DocumentIterator;
+import edu.illinois.cs.cogcomp.nlp.corpusreaders.ontonotes.utils.DocumentIterator.Language;
+import edu.illinois.cs.cogcomp.nlp.tokenizer.StatefulTokenizer;
+import edu.illinois.cs.cogcomp.nlp.utility.TokenizerTextAnnotationBuilder;
+
+/**
+ * convert the Ontonotes 5.0 format to CoNLL 2003 column format. Input arguments are
+ * required, the database url, the account name, the password, and the language, and
+ * finally the output file name.
+ * @author redman
+ */
+public class ConvertOntonotesToColumn {
+    static final private Set<String> tagsWithText = new HashSet<>();
+    static final private Set<String> dropTags = new HashSet<>();
+
+    // define the attributes we want to keep for the tags we have.
+    static final private Map<String, Set<String>> tagsWithAtts = new HashMap<>();
+    static {
+        Set<String> docAttrs = new HashSet<>();
+        docAttrs.add("docno");
+        tagsWithAtts.put("doc", docAttrs);
+        Set<String> nameAttrs = new HashSet<>();
+        nameAttrs.add("type");
+        tagsWithAtts.put("enamex", nameAttrs);
+    }
+
+    /**
+     * read the file indicated by the argument which is the file name, and path.
+     * @param document the data read from the file.
+     * @return the XmlTextAnnotation containing the text annotation, and xml markup offset data.
+     * @throws IOException 
+     */
+    static private XmlTextAnnotation getNameTextAnnotation(File file) throws IOException {
+        String document = LineIO.slurp(file.getCanonicalPath());
+        // we keep everything.
+        XmlDocumentProcessor xmlProcessor = new XmlDocumentProcessor(tagsWithText, tagsWithAtts, dropTags, true);
+        StatefulTokenizer st = new StatefulTokenizer();
+        TokenizerTextAnnotationBuilder taBuilder = new TokenizerTextAnnotationBuilder(st);
+        XmlTextAnnotationMaker xtam = new XmlTextAnnotationMaker(taBuilder, xmlProcessor);
+ 
+        // read the file and create the annotation.
+        XmlTextAnnotation xta = xtam.createTextAnnotation(document, "OntoNotes 5.0", "test");
+        TextAnnotation ta = xta.getTextAnnotation();
+        List<SpanInfo> fudge = xta.getXmlMarkup();
+ 
+        // create the named entity vi
```

Comment:
Yes, this behavior still exists, if I am correct wrt what you refer to here. It does not support nested entities. I will create an issue.